aws/eks/outputs.tf

```diff
@@ -25,3 +25,7 @@ output "subnets" {
 output "vpc_id" {
   value = module.eks-vpc.vpc_id
 }
+
+output "vpc" {
+  value = module.eks-vpc
+}
```

Comment:
If we’re exposing the entire module as an output, can we get rid of the `vpc_id` output above since it would now be available as `vpc.id`?